CHANGELOG.md

```diff
@@ -7,6 +7,11 @@
 - Use `canonicalName` in Fragment Integration for better de-obfuscation ([#2379](https://github.com/getsentry/sentry-java/pull/2379))
 - Fix Timber and Fragment integrations auto-installation for obfuscated builds ([#2379](https://github.com/getsentry/sentry-java/pull/2379))
 - Don't attach screenshots to events from Hybrid SDKs ([#2360](https://github.com/getsentry/sentry-java/pull/2360))
+- Do not attach empty `sentry-trace` and `baggage` headers ([#2385](https://github.com/getsentry/sentry-java/pull/2385))
+
+### Features
+
+- Add support for Opentelemetry ([#2344](https://github.com/getsentry/sentry-java/pull/2344))
```

Comment:
do you want to expand on this a bit more? e.g. link to the docs or just explaining briefly what it does

Reply:
Added readme and linked to it from changelog